src/mvAppItem.cpp

```diff
@@ -3947,18 +4056,20 @@ DearPyGui::GetEntityParser(mvAppItemType type)
             MV_PARSER_ARG_SHOW)
         );
 
-        args.push_back({ mvPyDataType::DoubleList, "x" });
-        args.push_back({ mvPyDataType::DoubleList, "y" });
-        args.push_back({ mvPyDataType::DoubleList, "negative" });
-        args.push_back({ mvPyDataType::DoubleList, "positive" });
-        args.push_back({ mvPyDataType::Bool, "contribute_to_bounds", mvArgType::KEYWORD_ARG, "True" });
-        args.push_back({ mvPyDataType::Bool, "horizontal", mvArgType::KEYWORD_ARG, "False" });
-
-        setup.about = "Adds an error series to a plot.";
+        args.push_back({ mvPyDataType::DoubleList, "values" });
+        args.push_back({ mvPyDataType::StringList, "label_ids" });
+        args.push_back({ mvPyDataType::Integer, "item_count" });
+        args.push_back({ mvPyDataType::Integer, "group_count"});
+        args.push_back({ mvPyDataType::Float, "group_size", mvArgType::KEYWORD_ARG, "0.67", "Size of bar groups" });
```

Comment:
`group_size`: I'd suggest to name this argument `group_width` instead. First, it is semantically closer to what the arg actually controls (relative group width), and second, it lets us use the name `group_size` for the number of bars per group.